src/components/Button/Button.tsx

```diff
@@ -125,64 +122,81 @@ const ButtonWithHandlers = React.forwardRef<HTMLElement, ButtonProps>(function B
                     view,
                 },
             });
+
+            if (onClickCapture) {
+                onClickCapture(event as any);
```

Comment:
Why is type cast used?

Reply:
It requires weird type, intersection of `HTMLButtonElement` and `HTMLAnchorElement`